src/main/java/liuni/TwitterConfig.java

```diff
@@ -0,0 +1,91 @@
+package liuni;
+
+import org.hibernate.validator.constraints.NotEmpty;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.Map;
+import twitter4j.Twitter;
+import twitter4j.TwitterFactory;
+import twitter4j.conf.ConfigurationBuilder;
+
+import javax.validation.constraints.NotNull;
+
+public class TwitterConfig extends io.dropwizard.Configuration {
+    @NotNull
+    private Map<String, String> twitter;
+
+    @NotEmpty
```

Comment:
why is this marked NotEmpty?

Reply:
Because the key names should not be empty?